eos-usb-reflash/factory-test/start.sh

```diff
@@ -6,6 +6,30 @@
 set -e
 mydir=$(dirname $(readlink -f $0))
 
+# If this script is running from an exFAT filesystem we need to manually load
+# the exfat kernel module, as on EOS prior to 4.0 the USB device has been
+# mounted using the FUSE exFAT driver, and the initramfs we are going to pivot
+# to does not contain an exFAT driver (neither in-kernel nor via FUSE).
+FSTYPES=$(lsblk --noheadings --output FSTYPE)
+if [[ "$FSTYPES" =~ "exfat" ]] ; then
+    echo "Detected an exFAT filesystem attached to this machine;"
+    echo "Trying to load the 'exfat' kernel module..."
+    set +e
+    modprobe exfat
+    exfat_loaded=$?
+    if [ $exfat_loaded -ne 0 ] ; then
```

Comment:
You don't need to `set +e`; you can use:

```
if ! modprobe exfat ; then
...
fi
```

Reply:
Good point!